### ISSUE TITLE
Suggestion: add the thumbnail list at the bottom like in Picasa

### ISSUE THREAD

**Issue by ccrowles** [riyasy/FlyPhotos#15]
As per title, one of the features I loved the most about Picasa Photo Viewer was having a thumbnail list at the bottom that would auto-hide and only show when moving the mouse.
I was unable to find that feature anywhere else, and currently FlyPhotos doesn't support it (but I deeply appreciate your willingness to bring a spiritual successor to such a great photo viewer).

To accomodate it properly, considering that right now FlyPhotos opens the images in fullscreen, taking up the entire space (overlapping with the buttons and even not showing the minimize/enlarge/close buttons), it would be nice if the thumbnail list option, when enabled, would reduce the default zoom size to make it not overlap the buttons and the thumbnail list, basically exactly like in Picasa. This would also provide space to show the minimize/enlarge/close buttons when in fullscreen (or at the very least the close button like in PPV).

Basically I would like for FlyPhotos to look and behave more like Picasa Photo Viewer, since it aims to be the ideal replacement of it.

Comparison with the same image in Picasa Photo Viewer when showing the thumbnail list (with a nice, comfortable space all around that makes very visible everything: thumbnail list, image, close button at the top right):

![Screenshot_2024-08-22_10-23-26_859](https://github.com/user-attachments/assets/519e754f-70b1-4763-8109-9956abe621a8)

and FlyPhotos:

![Screenshot_2024-08-22_10-44-59_212](https://github.com/user-attachments/assets/bc332653-3ee6-4baa-a561-a959f8d609dd)

**Comment by riyasy** [comment]
![image](https://github.com/user-attachments/assets/ce7cf4c6-8f55-409c-8f0e-515bb0fa8ee1)

Thanks for trying out and suggesting features and finding bugs.

I have implemented the thumbnail (not yet published). But it is read only means you cant click and navigate using thumbnail strip. The thing is Flyphotos photo display is done using directx (direct2d) and everything is rendered immediate mode. Otherwise we will never be able to develop half a fast a GUI as using the current WinUI development framework. To interact with the thumbnail and animate the movement of the thumbnail, it will require lot more coding in future. I am in a dilemma on whether to release the non-interactable thumbnail strip or wait till a fully interaction possible thumbnail strip is ready.

Regarding default fitment of the image with the suggested spacing, may need to wait for few more iterations.

**Comment by ccrowles** [comment]
Personally I would love even a non interactable version. If in the future a fully interactable one will be possible, that's even better, but I'd still rather have a read-only one than none at all.

Thank you for taking into consideration my suggestions, I don't mind waiting for the fitment with the spacing! But would it at be possible to add a close (x) button in the top-right soon-ish? Because currently the only way to close is hitting ESC on the keyboard, which is not always the most comfortable solution.

**Comment by riyasy** [comment]
Close button can be considered in full screen view.

**Comment by ccrowles** [comment]
Thank you! 💜

**Comment by riyasy** [comment]
Close button had been added in full-screen in 2.3.1

**Comment by riyasy** [comment]
Thumbnail interaction is possible now
